python/langsmith/client.py

```diff
@@ -3725,11 +3729,28 @@ def list_examples(
             params["dataset"] = dataset_id
         else:
             pass
+        if get_attachments == True:
+            params['select'] = ['attachment_urls', 'outputs', 'metadata']
         for i, example in enumerate(
             self._get_paginated_list("/examples", params=params)
         ):
+            attachments = {}
+            if example['attachment_urls']:
+                for key, value in example['attachment_urls'].items():
+                    def create_reader(url=value['presigned_url']):
+                        response = requests.get(url, stream=True)
+                        response.raise_for_status()
+                        # Return the raw response object so caller can handle streaming
+                        return response
```

Comment:
would move this outside the function/loop and make prefix with an underscore for readability

Reply:
have you tried using the reader object?

Reply:
(don't have to read obvsiouly, just return the reader)